Userland/Applications/Maps/MapWidget.cpp

```diff
@@ -171,6 +174,43 @@ void MapWidget::mousewheel_event(GUI::MouseEvent& event)
     set_zoom_for_mouse_event(new_zoom, event);
 }
 
+void MapWidget::context_menu_event(GUI::ContextMenuEvent& event)
+{
+    if (!m_context_menu_enabled)
+        return;
+
+    LatLng latlng = { tile_y_to_latitude(latitude_to_tile_y(m_center.latitude, m_zoom) + static_cast<double>(event.position().y() - height() / 2) / TILE_SIZE, m_zoom),
+        tile_x_to_longitude(longitude_to_tile_x(m_center.longitude, m_zoom) + static_cast<double>(event.position().x() - width() / 2) / TILE_SIZE, m_zoom) };
+
+    m_context_menu = GUI::Menu::construct();
+    m_context_menu->add_action(GUI::Action::create(
+        DeprecatedString::formatted("{:.5}, {:.5}", latlng.latitude, latlng.longitude), MUST(Gfx::Bitmap::load_from_file("/res/icons/16x16/edit-copy.png"sv)), [latlng](auto&) {
```

Comment:
Would it be more clear what this action is for if the text was `"Copy Coordinates to Clipboard" or some such?

Reply:
You are right I had copied that text from Google Maps and it was a bit vague. I have update the menu item label to "Copy Coordinates to Clipboard" as you suggested.